esx_service/vmdk_ops.py

```diff
@@ -1703,6 +1704,101 @@ def wait_for_tasks(si, tasks):
 
 #------------------------
 
+def start_vm_changelistener():
+    """
```

Comment:
How is this service notified to stop and exit itself, if that is needed at all? Say upgrade is done and service is stopped and restarted.